docs/csharp/expression-trees-summary.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Expression Trees Summary | Microsoft Docs
-description: Expression Trees Summary
+description: Recap how you can use expression trees to create dynamic programs that interpret code as data and build new functionality based on that code.
```

Comment:
oh ... I almost did this, but they want "[You] X" (action by user)-style descriptions, so the verb would be "recap" (i.e., "You recap ...").